oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCUserManager.java

```diff
@@ -204,14 +204,14 @@ private int sendLogout()
         try {
             logoutURI = this.configuration.getLogoutOIDCEndpoint();
         } catch (Exception e) {
-            // TODO: log something ?
+            this.logger.error("No OIDC logout URI set", e);
```

Comment:
That's actually not what this exception means, if no logout URI is set (which is not a problem at all) then this method will simply return null and not throw an exception.